x/deployment/types/event.go

```diff
@@ -39,6 +42,7 @@ func (ev EventDeploymentCreated) ToSDKEvent() sdk.Event {
 		append([]sdk.Attribute{
 			sdk.NewAttribute(sdk.AttributeKeyModule, ModuleName),
 			sdk.NewAttribute(sdk.AttributeKeyAction, evActionDeploymentCreated),
+			sdk.NewAttribute(evVersionKey, string(ev.Version)),
```

Comment:
this is going to be quite ugly.  let's make a type that (un)marshals hex or base36 or something - or do that by hand here.

Reply:
Updated to encode data as hex before writing as attribute.